sdk/communication/Azure.Communication.Calling.Server/src/Models/Events/CallingServerEventBase.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.Calling.Server
+{
+    /// <summary>
+    /// Base type for all calling events.
+    /// </summary>
+    public abstract class CallingServerEventBase
```

Comment:
Curious why we need an empty abstract class here? Will this contain content in the future?

Reply:
Added common type for events to support polymorphism.

Reply:
this is for Polymorphism. it will be easier on the client end to handle all events as same type